azure-pipelines.yml

```diff
@@ -50,29 +50,42 @@ jobs:
                        ripgrep pyflakes beautifulsoup4 chardet pycrypto glob2 psutil pytz tqdm  \
                        conda-package-handling py-lief python-libarchive-c perl conda-forge::shellcheck
       conda install -q pytest pip pytest-cov pytest-forked pytest-xdist nomkl numpy mock pytest-mock
+      conda install -q pytest-replay pytest-rerunfailures -c conda-forge
```

Comment:
@msarahan 
``pyteste-rerunfailures`` would be nice to have on the main channels as well 
I add it to rerun some tests which were failing because of loss of connection to the ``cran`` repositories
I think that is ready to be integrated, just need someone to review it :)